packages/universal-components/src/RangeDatePicker/RangeDatePicker.js

```diff
@@ -45,34 +72,48 @@ export default class RangeDatePicker extends React.Component<Props> {
         onBackdropPress={this.handleDismiss}
       >
         <View style={styles.content}>
-          <RangeDatePickerContent
-            selectedDates={this.props.dates}
-            onDayPress={this.handleChangeDate}
-            numberOfRenderedMonths={numberOfRenderedMonths}
-            weekStartsOn={weekStartsOn}
-            isRangePicker={isRangePicker ?? true}
-          />
+          {!isNightsInDestinationSelected || !isNightsInDestinationVisible ? (
+            <RangeDatePickerContent
+              selectedDates={this.props.dates}
+              onDayPress={this.handleChangeDate}
+              numberOfRenderedMonths={numberOfRenderedMonths}
+              weekStartsOn={weekStartsOn}
+              isRangePicker={isRangePicker ?? true}
```

Comment:
Can this prop be false if this component is called `RangeDatePicker`?

Reply:
I see :D 

If:
```
isRangePicker == true // it's possible to choose date range 
isRangePicker == false // it's possible to choose only one date (will be used for Hotels in Kiwi app)
```

Maybe there should be different naming and the best would be to rename whole `RangeDatePicker` to `DatePicker`, but it should be different PR. And I'm also not convinced if it worth for it. And there is also different `DatePicker` in `universal-components`

Maybe we could just rename this prop. For example `isSingleDateOnly`. What do you think?

Reply:
Ok, thanks for explanation. I don't think that component renaming is necessary.
Just changing prop should be fine. We can have `type` like in case of `Slider`. Or something like you proposed is also fine.
Adding storybook with playground to these new components later will also help. So user will be able to try what each prop actually does.

Reply:
Yes, totally agree. There is already issue for adding RangeDatePicker to storybook (#749). I will focus on it after landing page, which has right now bigger priority.